composer.json

```diff
@@ -28,7 +24,7 @@
     "symfony/finder": "^2.8",
     "symfony/yaml": "^2.8",
     "guzzlehttp/guzzle": "^6",
-    "symbiote/silverstripe-queuedjobs": "dev-queue-buildtask as 3.x-dev",
+    "symbiote/silverstripe-queuedjobs": "3.x-dev as 3.1.0",
```

Comment:
I've updated the branch alias on queuejobs so you can use 3.1.x-dev now without aliasing it